### DISCUSSION TITLE
Conditional profiles: any working examples

### DISCUSSION THREAD

**Discussion by richardsprague** [quarto-dev/quarto-cli#5222] [Q&A]
I'd like my project to generate both a book and a website, but I'm unable to modify  the simple examples in the documentation to do what I want.

I'd like to be able to type either
```
quarto render --profile mybook
```
or
```
quarto render --profile mywebsite
```

and have it do the right thing.

I tried making a `_quarto.yml` and then a `_quarto-mybook.yml` but maybe I'm just unclear from the documentation how I specify that I have multiple profiles.

I did a quick search of Github and couldn't find cases that resemble this, which is odd because it seems like a common scenario.

Any hints?

**Comment by mcanouil** [comment]
What exactly is unclear in <https://quarto.org/docs/projects/profiles.html#examples>?

Another simple example from Andrew Heiss: <https://github.com/andrewheiss/quarto-profiles-example>.
